.github/workflows/pr-title.yml

```diff
@@ -0,0 +1,28 @@
+name: Lint pull request title
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+permissions:
+  contents: read
+
+jobs:
+  main:
+    permissions:
+      pull-requests: read
+      statuses: write
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@5.5.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          scopes: |
+            foreman
+            deps
```

Comment:
Most of the time I find myself adding a scope that was missing entirely (e.g., #28, #16, #7). We'll definitely need to enforce a proper scope, maybe via `requireScope: true`. However, that would necessitate having a catch-all scope for changes to non-chart stuff (perhaps `meta`, or `repo`, or `main`, ...).